qiling/loader/elf.py

```diff
@@ -97,7 +101,7 @@ def run(self):
 
         # is it a driver?
         if elftype == 'ET_REL':
-            self.load_driver(elffile, stack_address + stack_size, loadbase=0x8000000)
+            self.load_driver(elffile, stack_address + stack_size, loadbase=0x1000000)
```

Comment:
Any reason for this change here?
Keep in mind this module serves all ELF files on all architectures, not only ARM.